wgpu-hal/src/metal/adapter.rs

```diff
@@ -91,6 +100,8 @@ impl crate::Adapter for super::Adapter {
             MTLReadWriteTextureTier::TierNone => (Tfc::empty(), Tfc::empty()),
             MTLReadWriteTextureTier::Tier1 => (Tfc::STORAGE_READ_WRITE, Tfc::empty()),
             MTLReadWriteTextureTier::Tier2 => (Tfc::STORAGE_READ_WRITE, Tfc::STORAGE_READ_WRITE),
+            // Fall back to `MTLReadWriteTextureTier2`
```

Comment:
nitpick: Can we note that this behavior feels appropriate because it's likely a _higher_ level of support, please?